wallet/wallet.go

```diff
@@ -33,11 +29,12 @@ func NewWallet() (wallet Wallet, err error) {
 	if err != nil {
 		return
 	}
-	return Wallet{Account: newAccount}, nil
+	signer := blswallet.BLSSigner{Account:newAccount, Domain: defaultDomain}
```

Comment:
In the future, we'll need to use the correct domain here instead of defaultDomain. Created an issue for this: #72.